index.js

```diff
@@ -29,58 +29,92 @@ const availableCommandsMessage = `Available commands are:-
 	/* --------------------------------- ANCHOR Automation --------------------------------- */
-	app.on('pull_request.opened', addReadyForReviewLabel);
+	app.on("pull_request.opened", addReadyForReviewLabel);
 
-	app.on('pull_request_review.submitted', addApprovedLabel);
-	app.on('pull_request_review.submitted', changesRequestLabel);
+	app.on("pull_request_review.submitted", addApprovedLabel);
+	app.on("pull_request_review.submitted", changesRequestLabel);
 
-	app.on('pull_request.closed', addMergedLabel);
+	app.on("pull_request.closed", addMergedLabel);
+	app.on("pull_request.closed", async context => {
+		const params = context.pullRequest();
+
+		try {
+			const pullRequestIsMerged = await context.octokit.pulls.checkIfMerged(
+				params
+			);
+		} catch (err) {
+			const issueLabels = await listIssueLabels(context);
+			const foundReadyForReviewLabel = issueLabels.data.filter(
+				label => label.name === ":mag: Ready for Review"
+			);
+			const foundApprovedLabel = issueLabels.data.filter(
+				label => label.name === ":white_check_mark: Approved"
+			);
+			const foundChangesRequestedLabel = issueLabels.data.filter(
+				label => label.name === ":warning: Changes requested"
+			);
+
+			if (foundReadyForReviewLabel.length > 0) {
+				removeLabel([":mag: Ready for Review"], context);
+			}
+
+			if (foundApprovedLabel.length > 0) {
+				removeLabel([":white_check_mark: Approved"], context);
+			}
+
+			if (foundChangesRequestedLabel.length > 0) {
+				removeLabel([":warning: Changes requested"], context);
+			}
+
+			addLabel([":x: closed"], context, "B60205");
+		}
+	});
 
 	app.on(
-		['pull_request.edited', 'pull_request.labeled'],
+		["pull_request.edited", "pull_request.labeled"],
 		pullRequestWIPLabelAutomation
 	);
 
-	app.on('issues.closed', addLabelToIssueOnClose);
+	app.on("issues.closed", addLabelToIssueOnClose);
 
 	/* --------------------------------- ANCHOR Issue commands --------------------------------- */
-	app.on('issue_comment.created', async context => {
+	app.on("issue_comment.created", async context => {
 		const { body } = context.payload.comment;
 		const { command, args } = getCommandAndArgs(body);
 		const commentId = context.payload.comment.id;
 
-		if (command[0] === '/' && !context.isBot) {
+		if (command[0] === "/" && !context.isBot) {
 			const params = context.issue({
 				comment_id: commentId,
-				content: '+1',
+				content: "+1",
 			});
 
 			context.octokit.reactions.createForIssueComment(params);
 		}
 
 		switch (command) {
-			case '/label':
+			case "/label":
 				labelCommand(context, args);
 				break;
-			case '/close':
+			case "/close":
 				closeCommand(context);
 				break;
-			case '/approve':
+			case "/approve":
 				approveCommand(context);
 				break;
-			case '/merge':
+			case "/merge":
 				mergeCommand(context);
 				break;
-			case '/WIP':
+			case "/WIP":
 				WIPCommand(context);
 				break;
 
 			default:
-				if (command[0] === '/') {
+				if (command[0] === "/") {
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹↹` with `········`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060881)
```suggestion
        if (command[0] === "/") {
```